src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -132,25 +130,55 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             },
             handleError: "{that}.events.onError.fire",
             handlePause: {
-                changePath: "paused",
-                value: true
+                funcName: "fluid.textToSpeech.handlePause",
+                args: ["{that}"]
             },
             handleResume: {
-                changePath: "paused",
-                value: false
+                funcName: "fluid.textToSpeech.handleResume",
+                args: ["{that}"]
             }
         }
     });
 
+    // Issue commands to the speechSynthesis interface after a delay; this
+    // makes the wrapper behave somewhat better when issuing commands, especially
+    // play and pause
+    fluid.textToSpeech.asyncSpeechSynthesisControl = function (control, delay, args) {
+        setTimeout(function () {
+            speechSynthesis[control](args);
+        }, delay);
+    };
+
     fluid.textToSpeech.speak = function (that, speaking) {
         that.events[speaking ? "onStart" : "onStop"].fire();
     };
 
-    fluid.textToSpeech.pause = function (that, paused) {
-        if (paused) {
-            that.events.onPause.fire();
-        } else if (that.model.speaking) {
-            that.events.onResume.fire();
+    fluid.textToSpeech.handlePause = function (that) {
+        that.applier.change("paused", true);
+        that.events.onPause.fire();
```

Comment:
Point taken - but in this case, we do need to establish some level of communication and sequencing between the wrapper component and the WebSpeechSynthesis API (so that the model doesn't enter a state that doesn't actually reflect what's happening with the browser API, which is where some of the problems have centred working on the issue of irregularly failing tests). But I'll look for ways to simplify.